pyro/distributions/util.py

```diff
@@ -199,18 +199,16 @@ def matrix_triangular_solve_compat(b, A, upper=True):
         return b.trtrs(A, upper=upper)[0].view(b.shape)
 
 
-def log_sum_exp(tensor, dim=-1, scale=1.0):
+def log_sum_exp(tensor, dim=-1):
```

Comment:
Removing the scale argument, which does not work when scale has negative values (which will be the case since these are just samples in our use case) and the resulting log will give NaNs. The `sum_exp` computation is moved to `Empirical._weighted_mean()`.